modules/ROOT/pages/import-api-task.adoc

```diff
@@ -1,13 +1,19 @@
-= Creating an API instance by importing on API Manager
+= Creating an API Instance by Importing to API Manager
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can develop an API for and export it from API Manager for future use or for sharing. The exported API can be imported using this procedure. To perform this task, you need to have a ZIP file on your file system containing an API object exported from API Manager.
+You might have developed an API and exported it from API Manager to share it across your organization or to use it at a later time. You can then import this API into API Manager when required. 
```

Comment:
```suggestion
Imagine that you develop an API that you export from API Manager so that you can share it across your organization or use it later. You can still import this API back into API Manager, as required. 
```
I'm not sure that I understood this correctly. 
Is the point that we are trying to make that after you create an API in API Manager, you can export it to be used elsewhere, but you can also later retrieve it? 
I get confused at "or to use it at a later time"; what does that mean, that you export it and save it somewhere outside of API Manager? Why would you do that? Under what circumstances?
And then why would you have to return it to API Manager?
So, this topic isn't really about creating by import, but about returning from export?
Consider running this intro para by another writer and see what they have to offer about clarifying the wording, because I don't understand what you are trying to say.

Reply:
Fixed as I saw fit. Thanks!